e2etests/test/sas_token_tests.js

```diff
@@ -12,9 +12,32 @@ var Client = require('azure-iot-device').Client;
 var Message = require('azure-iot-common').Message;
 var SharedAccessSignature = require('azure-iot-device').SharedAccessSignature;
 var ConnectionString = require('azure-iot-device').ConnectionString;
+var DeviceIdentityHelper = require('./device_identity_helper.js');
+
+var hubConnectionString = process.env.IOTHUB_CONNECTION_STRING;
+var transports  = [
+  require('azure-iot-device-amqp').Amqp,
+  require('azure-iot-device-amqp').AmqpWs,
+  require('azure-iot-device-mqtt').Mqtt,
+  require('azure-iot-device-mqtt').MqttWs,
```

Comment:
insignificant, but does this load and parse the MQTT module twice?

Reply:
no. require() caches the module once it's loaded. 

---
In reply to: [149450212](https://github.com/Azure/azure-iot-sdk-node/pull/134#discussion_r149450212) [](ancestors = 149450212)